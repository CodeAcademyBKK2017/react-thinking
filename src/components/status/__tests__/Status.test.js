import React from 'react';
import renderer from 'react-test-renderer';
import {shallow} from 'enzyme';
import Status from '../Status.component';

describe('Status component', () => {
  it('Status: renders correctly', () => {
    const tree = renderer.create(<Status />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
