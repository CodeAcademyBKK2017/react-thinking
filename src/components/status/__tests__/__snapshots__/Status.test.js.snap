// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Status component Status: renders correctly 1`] = `
<input
  type="text"
  value={0}
/>
`;
